examples/i18n/app.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div>
-    Hi from @nuxtjs/i18n: {{ (('hello')) }}
+    Hi from @nuxtjs/i18n: {{ $t('hello') }}
+  </div>
+  <div>
```

Comment:
```suggestion
  <i18n-t scope="global" keypath="message"><template #module>@nuxtjs/i18n</template></i18n-t>
  <div>
```
The error in i18n _can_ be fixed by importing `i18n` from `useNuxtApp().$i18n`. However, I couldn't find a way to fix the `i18n-t` component. I tried replacing it with the `Translation` component imported from `vue-i18n`, but the error persists.

This suggested change will also need an addition to the `i18n` messages:
`"message": "Hello from {module}"`